ui/app/components/clients/charts/line.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import Component from '@glimmer/component';
-import { SVG_DIMENSIONS, formatNumbers } from 'vault/utils/chart-helpers';
+import { SVG_DIMENSIONS, numericalAxisLabel } from 'vault/utils/chart-helpers';
```

Comment:
renamed to differentiate between `formatNumber` the template helper and the chart axis formatter